app/services/waste_carriers_engine/govpay_update_refund_status_service.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module WasteCarriersEngine
+  class GovpayUpdateRefundStatusService < WasteCarriersEngine::BaseService
+
+    def run(registration:, refund_id:, new_status:)
+      return false if new_status != "success"
```

Comment:
minor: it might be a good idea to store these statuses as registration class constants (or payment/refund class constants if we have such dedicated classes) 

e.g. 

`new_status != WasteCarriersEngine::Payment::PAYMENT_STATUS_SUCCESS`